test/e2e/kubectl.go

```diff
@@ -70,6 +70,7 @@ const (
 	simplePodName            = "nginx"
 	nginxDefaultOutput       = "Welcome to nginx!"
 	simplePodPort            = 80
+	runJobTimeout            = 5 * time.Minute
```

Comment:
was concerned that foreverTestTimeout might be too short. and there are other timeouts in this file as well, so this seemed consistent